### ISSUE TITLE
[Bug] sync stops working

### ISSUE THREAD

**Issue by DavidHaltinner** [mozilla-mobile/fenix#4196]
## Steps to reproduce
Use this preview, normal android firefox (68), and linux desktop Firefox(67 or 68 verifed), each utilizing the same Firefox account

### Expected behavior
Sync is the same between each browser.

### Actual behavior
Classic android and desktop versions sync bookmarks (i sync bookmarks and add ons only, though add ons arent supported here. I dont sync anything else in other browsers, but bonus note: this preview wont aloow me to uncheck history sync), but the preview version does not sync changes that occur in the other browsers, only changes that occured within itself.
### Device information

* Android device: 
LineageOS 16.0 (july 8th build for Shamu as of this posting, though did occur on other daily builds as well)

* Fenix version: ?
1.00, 1.0.1 and 1.1.0rc1 as of this posting

**Comment by boek** [comment]
@DavidHaltinner To clarify if you make a bookmark on the desktop version of Firefox it isn't syncing to Firefox Preview?

**Comment by DavidHaltinner** [comment]
Well, I played with it a bit more after posting, and am more confused myself.
It seems that its more specifically changes or removals of bookmarks that are causing strange behviour, but not all the time.

So I go into desktop, make a bookmark, sync, open Preview, sync, and the bookmark shows in both browsers. But if I move the bookmark to a folder, or remove it in desktop, it appears that most of the time, but not every time, that change is picked up by Preview. For instance, I have a bookmark that was made, and a few days later removed, but preview is still showing it in the list, no other browser shows it in their list, only Preview does. But if I go in right now and re-do the steps for another page like amazon.com, it works as should be expected.
So at the moment I just have one odd orphan bookmark in Preview no one else sees. I did not remove it manually from Preview yet, I assume it will allow me to just fine, but in case there was anything to check.

Not sure if it helps, but the orphan bookmark I have left right now, that only Preview sees is to this page on the raspberry pi site, just clicked the star icon, did not customize the bookmark in any way:  https://www.raspberrypi.org/blog/raspberry-pi-4-on-sale-now-from-35/

Not sure it
